CONTRIBUTING.md

```diff
@@ -36,7 +36,15 @@ Before we're able to merge your code into the ownCloud app for Android, please,
 	Please, use the mentioned prefixes because CI system is ready to match with them. Be sure your feature, fix, improvement or technical branches are updated with latest changes in official `android/master`, it will give us a better chance to test your code before merging it with stable code.
 * Once you are done with your code, start a pull request to merge your contribution into official `android/master`.
 * Keep on using pull requests for your next contributions although you own write permissions.
-* Important to mention that ownCloud Android team uses [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) as branching model. Please take a look to the link to understand better what it is and how it works. It's something as useful as easy.
+* Important to mention that ownCloud Android team uses OneFlow. as branching model. It's something as useful as easy:
+
+  * `master` will stay as main branch. Everything will work around it.
+  * Feature branch: new branch created from `master`. Once it is finished and DoD accomplished, rebased and merged into `master`.
+  * Release branch: will work as any feature branch. Before rebasing and merging into `master`, release tag must be signed.
+  * Hotfix branch: created from latest tag. Once it is finished, tag must be signed. Then, rebased and merged into `master`.
+  * The way to get an specific version is browsing through the tags.
```

Comment:
Idk if we talk about tags in some other point of the documentation, but would be interesting mentioning that it requires a separate `fetch` to update them, and including the command used to do it. What do you think?

Reply:
Not sure about this, because it's an specific detail of the git use. We should guide users through the process, not teaching how to use an external (involved) tool beyond the steps to handle the repo. I get your point, but also, i don't know in which point i'd add that.

Reply:
In this same document, in section `2. Create pull request` for example, we include several git commands to help the users. It's true that this is not totally part of the process but would be good to know in case someone wants to work among different versions. Not sure where  and if including it either 🤷🏼‍♂️

Reply:
Added a new section in `CONTRIBUTING.md` file about "Versioning" . What do you think?